helpers/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "litesvm-helpers"
```

Comment:
let's rename this to `litesvm-token` or something. `litesvm-helpers` could mean anything

Reply:
It's not only for tokens, it will be for Pyth, loader, etc..

Reply:
Is there a good, precise name that describes what this crate will be? If not I think the intended contents of this crate should instead be split across multiple crates

> It's not only for tokens, it will be for Pyth, loader, etc..